recipes/transabyss/meta.yaml

```diff
@@ -9,24 +9,24 @@ build:
 
 package:
   name: transabyss
-  version: '1.5.5'
+  version: '2.0.1'
 
 requirements:
   host:
     - python
   run:
     - zlib
```

Comment:
why is zlib needed if we do not compile anything, same for libiconv

Reply:
No idea. I am not a maintainer of this code, just a user. 

Happy to remove these and see how things go.